documentation/book/proc-configuring-kafka-brokers.adoc

```diff
@@ -5,15 +5,18 @@
 [id='proc-configuring-kafka-brokers-{context}']
 = Configuring Kafka brokers
 
+You can configure an existing Kafka broker, or create a new Kafka broker with a specified configuration.
+
 .Prerequisites
 
-* An {ProductPlatformName} cluster
-* A running Cluster Operator
+* An {ProductPlatformName} cluster exists.
+* The Cluster Operator is running.
 
 .Procedure
 
-. Edit the `config` property in the `Kafka` resource specifying the cluster deployment.
-For example:
+. Open the YAML configuration file that contains the `Kafka` resource specifying the cluster deployment.
+
+. Edit the `config` property in the `Kafka.spec.kafka` resource. For example:
```

Comment:
The resource is just `Kafka`. The rest of it ... `.spec.kafka` is the path in the hierarchy inside the resource.

Reply:
Helpful feedback, thank you. I've reworded this step in every procedure. Please take a look.